tests/WorkOS/WorkOSServiceProviderTest.php

```diff
@@ -6,18 +6,18 @@ class WorkOSServiceProviderTest extends LaravelTestCase
 {
     protected $app;
 
-    protected function setUp()
+    protected function setUp(): void
```

Comment:
🚨 issue: with this `PHP 7.1` feature, we should also upgrade composer php version, to 
```
"php": ">=7.1",
```